web/app.js

```diff
@@ -463,6 +464,7 @@ const PDFViewerApplication = {
       enablePermissions: AppOptions.get("enablePermissions"),
       pageColors,
       mlManager: this.mlManager,
+      abortController: this._globalAbortController,
```

Comment:
Do we need to provide the "entire" AbortController here, or would the signal suffice since that'd lead to shorter code in the viewer components?
```suggestion
      abortSignal: this._globalAbortController.signal,
```